optaplanner-bom/pom.xml

```diff
@@ -418,6 +418,12 @@
         <version>${version.org.kie}</version>
         <classifier>sources</classifier>
       </dependency>
+      <!--KIE Drools Workbench Playground -->
+      <dependency>
+        <groupId>org.kie.workbench.playground</groupId>
+        <artifactId>drools-playground-parent</artifactId>
```

Comment:
This will break the build because this new repository isn't added to the repository-list.txt